Build/start.sh

```diff
@@ -28,9 +27,10 @@ echo "    CONFIGURATION COPIED TO conf/"
 if [ -z $DLOAD ];
 then
 	cd $WORK/build
-	7z x ~/oss_pkg_v3.0.0.7z
+	7z x ~/oss_pkg_rzv_v3.0.0.7z
 fi
 ##Apply DRPAI patch
+echo "IMAGE_INSTALL_append = \" gstreamer1.0-drpai ai-eva-sw\"" >> ${WORK}/meta-mistysom/recipes-core/images/mistysom-image.bbappend
```

Comment:
This should not be added to the G2L repo. It is specific to V2L

Reply:
Got it! Of course!